tools/importer-rest-api-specs/components/testing/attribute_value_for_field.go

```diff
@@ -147,6 +147,13 @@ var attributeValuesForBasicTypes = map[resourcemanager.TerraformSchemaFieldType]
 			return &val, nil
 		}
 
+		for k, v := range testData.Strings {
+			if strings.EqualFold(field.HclName, k) {
```

Comment:
Double-checking: do we need the `EqualFold` here? At this point the input should be normalised (so if it's not then we've got a problem elsewhere)?